jira/client.py

```diff
@@ -1309,6 +1310,188 @@ def update_filter(
         raw_filter_json = json.loads(r.text)
         return Filter(self._options, self._session, raw=raw_filter_json)
 
+    # Teams
+
+    def create_team(
+        self,
+        org_id: str,
+        description: str,
+        display_name: str,
+        team_type: str,
+        site_id: str = None,
+    ) -> Team:
+        """Creates a team, and adds the requesting user as the initial member.
+
+        Args:
+            org_id (str): organization identifier
+            description (str): description field of the team to be created
+            display_name (str): name of the team to be created
+            team_type (str): either 'OPEN' or 'MEMBER_INVITE'
+            site_id (Optional[str])
+
+        Returns:
+            Team
+        """
+        url = f"gateway/api/public/teams/v1/org/{org_id}/teams/"
+        payload = {
+            "description": description,
+            "displayName": display_name,
+            "teamType": team_type,
+        }
+        if site_id is not None:
+            payload["siteId"] = site_id
+        r = self._session.post(url, data=json.dumps(payload))
+        raw_team_json: dict[str, Any] = json_loads(r)
+        return Team(self._options, self._session, raw=raw_team_json)
+
+    def get_team(self, org_id: str, team_id: str, site_id: str = None) -> Team:
+        """Get the specified team.
+
+        Args:
+            org_id (str): organization identifier
+            team_id (str): team identifier
+            site_id (Optional[str])
+
+        Returns:
+            Team
+        """
+        url = f"gateway/api/public/teams/v1/org/{org_id}/teams/{team_id}"
+        if site_id is not None:
+            url += f"?siteId={site_id}"
+        r = self._session.get(url)
+        raw_team_json: dict[str, Any] = json_loads(r)
+        return Team(self._options, self._session, raw=raw_team_json)
+
+    def remove_team(
+        self,
+        org_id: str,
+        team_id: str,
+    ):
+        """Delete the specified team.
+
+        Args:
+            org_id (str): organization identifier
+            team_id (str): team identifier
+
+        Returns:
+            bool
+        """
+        url = f"gateway/api/public/teams/v1/org/{org_id}/teams/{team_id}"
+        r = self._session.delete(url)
+        if r.status_code == 204:
+            return True
+        return False
```

Comment:
```suggestion
        return r.ok
```

Would be the preferred way if you just wanted a boolean